README.md

```diff
@@ -3,3 +3,38 @@
 This repository contains the `mpdev` tool supporting the development of VMs deployable via
 [Google Cloud Marketplace](https://console.cloud.google.com/marketplace).
 
+## Installation
+
+### Prerequisites
+
+The following tools must be installed before using `mpdev`.
+* [docker](https://docs.docker.com/get-docker/)
+* [gsutil](https://cloud.google.com/storage/docs/gsutil_install)
+
+### Download latest release
+
+https://github.com/GoogleCloudPlatform/marketplace-tools/releases/latest
+
+### Install with Homebrew
+
+```
+brew tap GoogleCloudPlatform/mpdev https://github.com/GoogleCloudPlatform/mpdev.git
```

Comment:
we don't have this repo, can you help explain? (I have no idea how brew is setup)

Reply:
This is my mistake, it should be `GoogleCloudPlatform/marketplace-tools`